makefile

```diff
@@ -163,7 +163,7 @@ endif
 
```

Comment:
Can you add a comment explaining what `make pretty-lines` does? Any reasonable and short explanation is fine.